src/explore-education-statistics-admin/src/pages/publication/components/PublicationInviteNewUsersTab.tsx

```diff
@@ -1,7 +1,114 @@
 import React from 'react';
+import { BasicPublicationDetails } from 'src/services/publicationService';
+import { ReleaseSummary } from '@admin/services/releaseService';
+import Yup from '@common/validation/yup';
+import { Form, Formik } from 'formik';
+import {
+  FormFieldCheckboxGroup,
+  FormFieldTextInput,
+} from '@common/components/form';
+import useFormSubmit from '@common/hooks/useFormSubmit';
+import ButtonGroup from '@common/components/ButtonGroup';
+import Button from '@common/components/Button';
+import userService from '@admin/services/userService';
+import { mapFieldErrors } from '@common/validation/serverValidations';
+import { useHistory } from 'react-router-dom';
+import { generatePath } from 'react-router';
+import {
+  publicationManageTeamAccessRoute,
+  PublicationRouteParams,
+} from '@admin/routes/routes';
 
-const PublicationInviteNewUsersTab = () => {
-  return <p>Invite new users tab content</p>;
+interface InviteContributorFormValues {
+  email: string;
+  releaseIds: string[];
+}
+
+export const errorMappings = [
+  mapFieldErrors<InviteContributorFormValues>({
+    target: 'releaseIds',
+    messages: {
+      NOT_ALL_RELEASES_BELONG_TO_PUBLICATION:
+        "Some of the releases don't belong to the publication",
+      USER_ALREADY_HAS_RELEASE_ROLE_INVITES:
+        'The user has already been invited with these permissions',
+      USER_ALREADY_HAS_RELEASE_ROLES: 'The user already has these permissions',
+    },
+  }),
+];
+
+export interface Props {
+  publication: BasicPublicationDetails;
+  releases: ReleaseSummary[];
+}
+
+const PublicationInviteNewUsersTab = ({ publication, releases }: Props) => {
```

Comment:
Seems like we don't have any tests for this tab?

Reply:
Created `PublicationInviteNewUsersTab.test.tsx`